### PR TITLE
GEXF importer/exporter

### PR DESCRIPTION
**Pull request by d-michail**
Added GEXF importer/exporter support.

----
- [x] I read and understood <https://github.com/jgrapht/jgrapht/wiki/Become-a-Contributor>
- [x] I read and understood <https://github.com/jgrapht/jgrapht/wiki/How-to-make-your-first-%28code%29-contribution>
- [x] I added [unit tests](https://github.com/jgrapht/jgrapht/wiki/Unit-testing)
- [x] I added [documentation](https://github.com/jgrapht/jgrapht/wiki/How-to-write-documentation)
- [x] I followed the [Coding and Style Conventions](https://github.com/jgrapht/jgrapht/wiki/Coding-and-Style-Conventions)
- [x] I **have not** modified `HISTORY.md` or `CONTRIBUTORS.md`
- [x] I ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)

### CODE REVIEW

File: jgrapht-io/src/main/java/org/jgrapht/nio/gexf/GEXFAttributeType.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        return name;
+    }
+
+    /**
+     * Create a type from a string representation
+     * 
+     * @param value the name of the type
+     * @return the attribute type
+     */
+    public static GEXFAttributeType create(String value)
```
Comment (jsichi) [inline jgrapht-io/src/main/java/org/jgrapht/nio/gexf/GEXFAttributeType.java:59]: Won't Enum.valueOf do this automatically?

File: jgrapht-io/src/main/java/org/jgrapht/nio/gexf/package-info.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/**
+ * Graph Exchange XML Format (GEXF) importers/exporters
+ */
+package org.jgrapht.nio.gexf;
```
Comment (jsichi) [inline jgrapht-io/src/main/java/org/jgrapht/nio/gexf/package-info.java:4]: Maybe add a link to the spec here?

File: jgrapht-io/src/main/java/org/jgrapht/nio/gexf/GEXFAttributeType.java (hunk 1 above)
Comment (d-michail) [reply jgrapht-io/src/main/java/org/jgrapht/nio/gexf/GEXFAttributeType.java:59]: You are probably right. But then I will have to convert to uppercase first. Similar case with the toString() as well. If I remove it, I will have to use as toString().toLowercase()

Comment (jsichi) [reply jgrapht-io/src/main/java/org/jgrapht/nio/gexf/GEXFAttributeType.java:59]: Fair enough.

**Comment by jsichi** [comment]
Thanks!

**Pull request merged**